ftl/core/deck-config.ftl

```diff
@@ -411,8 +411,9 @@ deck-config-reschedule-cards-warning =
     Use this option sparingly, as it will add a review entry to each of your cards, and
     increase the size of your collection.
 deck-config-ignore-before-tooltip-2 = 
-    If set, cards introduced before the provided date will be ignored when optimizing FSRS parameters.
+    If set, cards reviewed before the provided date will be ignored when optimizing FSRS parameters.
     This can be useful if you imported someone else's scheduling data, or have changed the way you use the answer buttons.
+    If all of your cards have been reviewed before the selected date and you do not plan to add new cards, it's better to use Reset.
```

Comment:
```suggestion
    If all of your cards have been reviewed before the selected date and you do not plan to add new cards, it's better to reset the cards to new.
```

How about that?

Reply:
Me and user1823 are still discussing what's the best course of action in this case (all cards reviewed before the selected date, no new cards will be added), but yeah, I don't mind this correction.